src/content/blog/2025-05-05-celebrating-ten-years-of-discover.mdx

```diff
@@ -13,33 +13,33 @@ cover_image_alt: ugrc social card
 
 ## So, where did Discover come from?
 
-Before we implemented the current Discover service, UGRC utilized a program called Image Server through ArcGIS. It was available for public and private use, but connections made to the service created a heavy digital load on the [ArcGIS Server](https://enterprise.arcgis.com/en/image/latest/get-started/windows/what-is-arcgis-image-server-.htm). This resulted in a number of problems for both UGRC and our user base; the service experienced frequent downtime and other issues. While this old service did provide significant customization options for imagery and topographic maps, it created more problems than it solved.
+Before we implemented the current Discover service, UGRC utilized a program called ArcGIS Image Server. It was available for public and private use, but connections made to the service created a heavy digital load on the [ArcGIS Server](https://enterprise.arcgis.com/en/image/latest/get-started/windows/what-is-arcgis-image-server-.htm). This resulted in a number of problems for both UGRC and our user base; the services experienced rendering latenecy and occasional downtime. While this old service did provide significant customization options for imagery and topographic maps, it was not the best solution.
 
-For a period of time, UGRC resolved Utah's need for high-resolution aerial imagery data through contracts with federal programs. These programs provided nationwide imagery every couple of years, but in 2012, those programs came to an end. Thus, we needed a new solution.
+For a period of time, UGRC resolved Utah's need for high-resolution aerial imagery through contracts with federal programs. These programs provided a patchwork of imagery every couple of years, but in 2012, those programs came to an end. Thus, we needed a new solution.
 
 ## From Giza to Discover
 
-Shortly after aerial imagery through federal programs was no longer available, UGRC began working with Google and [Sanborn](https://sanborn.com/) to create an appliance called Giza, which serves Google imagery as WMS and WMTS. UGRC was one of the earliest adopters of this new Google imagery program, especially among state geospatial offices. We developed a Utah-specific version of Giza and named it Discover. In 2015, Discover officially launched as Utah's premier source for aerial imagery and dynamic basemaps.
+Shortly after high resolution aerial imagery through federal programs was no longer available, UGRC began working with Google and [Sanborn](https://sanborn.com/) to create a tile pyrimid server appliance called Giza, which serves Google imagery and basemaps as WMS and WMTS. UGRC was one of the earliest adopters of this new Google imagery programs. We developed a Utah-specific version of Giza and named it Discover. In 2015, Discover officially launched as Utah's premier source for aerial imagery and basemaps services.
```

Comment:
this new Google imagery programs.

This needs to go to these or programs needs to go to program.